src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -359,9 +357,9 @@ export default class TutorialView {
             text: ['Once {CALLSIGN} has taken off, you\'ll notice it will climb to {INIT_ALT} by itself. This is one of the instructions ',
                     'we gave them when we cleared them "as filed". Aircraft perform better when they are able to climb directly',
                     'from the ground to their cruise altitude without leveling off, so let\'s keep them climbing! Click it and type "cvs" (for',
-                    '"climb via SID"). Then they will follow the altitudes and speeds defined in the {SID_NAME} departure',
-                    'procedure. Feel free to click the speedup button on the right side of the input box (it\'s two small arrows)',
-                    'to watch the departure climb along the SID. Then just click it again to return to 1x speed.'
+                    '"climb via SID"). Then they will follow the altitudes and speeds defined in the {SID_NAME} departure. You can also simply',
+                    'give a direct climb, lifting the restrictions on the SID. Feel free to click the speedup button on the right side of the ',
+                    'input box (it\'s two small arrows) to watch the departure climb along the SID. Then just click it again to return to 1x speed.'
             ].join(' '),
             parse: (t) => {
                 if (prop.aircraft.list.length <= 0) {
```

Comment:
The 'Departure destinations' step is quite out-dated (probably introduced around the time SIDs were fist implemented, but worked very differently to how they work now)

The 'Radar Vectors' step could include a sentence or two about the track measuring tool

The 'Speed' step shouldn't mention TRACON controllers specifically (these only exist in America AFAIK)
The 'speed/sp' command should also be mentioned I think

Reply:
Yeah, I think I'll sort of rewrite that description for SIDs ig.

I can't believe I forgot the track measuring tool I use it so much XD.

Yeah, 100% I'll include the sp command and remove TRACON, stuff I overlooked from the old tutorial!